internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/controller/VeniceParentHelixAdminTest.java

```diff
@@ -643,9 +643,9 @@ private void testAddValueSchemaDocUpdate(ControllerClient parentControllerClient
     String owner = "test_owner";
     String keySchemaStr = "\"long\"";
     String schemaStr =
-        "{\"type\":\"record\",\"name\":\"KeyRecord\",\"fields\":[{\"name\":\"name\",\"type\":\"string\",\"doc\":\"name field\"},{\"name\":\"id1\",\"type\":\"double\"}]}";
+        "{\"type\":\"record\",\"name\":\"KeyRecord\",\"fields\":[{\"name\":\"name\",\"type\":\"string\",\"doc\":\"name field\"},{\"name\":\"id1\",\"type\":\"double\", \"default\": 0.0}]}";
```

Comment:
It seems the unit test doesn't cover the new logic to validate the new behavior when write compute is first enabled.
Can you expand the test cases to cover both succeed/failure scenario with the new logic?

Reply:
Sure.